msdk-io-mgf/src/main/java/io/github/msdk/io/mgf/MgfFileImportMethod.java

```diff
@@ -0,0 +1,168 @@
+package io.github.msdk.io.mgf;
```

Comment:
Please add the license header to all .java files (copy & paste from existing files in MSDK)

Reply:
Ah sorry I see the license is there.. but it should be above the imports (license should go first).